OpenRA.Mods.Common/Traits/World/EditorResourceLayer.cs

```diff
@@ -219,8 +164,5 @@ public struct EditorCellContents
 		public static readonly EditorCellContents Empty = default(EditorCellContents);
 		public ResourceType Type;
 		public int Density;
-		public string Variant;
-		public ISpriteSequence Sequence;
-		public int Frame;
```

Comment:
With the removal of these properties we can use the `CellContents` struct and remove the `EditorCellContents` struct.

Reply:
Done.